### PR TITLE
🎅 Internationalization

### PR DESCRIPTION
**Pull request by vasuchal**
## Description
Translates app using react-i18next

Fixes #<number>

## Screenshots
Screen Shot 2021-05-01 at 3 41 53 PM

### CODE REVIEW

File: src/i18n.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  .init({
+    react: {
+      useSuspense: false,
+    },
+    fallbackLng: 'en',
+    interpolation: {
+      escapeValue: false,
+    },
+    backend: {
+      // for all available options read the backend's repository readme file
```
Comment (Qrtn) [inline src/i18n.js:19]: Can you remove this comment?

File: src/i18n.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+i18n
+  .use(initReactI18next) // passes i18n down to react-i18next
+  .use(Backend)
+  .use(LanguageDetector)
+  .init({
+    react: {
+      useSuspense: false,
+    },
+    fallbackLng: 'en',
```
Comment (Qrtn) [inline src/i18n.js:14]: add a comment here -- this is the default language if no language is detected by LanguageDetector

File: src/components/settings/SettingsPage.js [hunk 3]
```diff
@@ -85,25 +87,25 @@ const SettingsPage = ({ hideTitle }) => {
...
-      <h5>Map View</h5>
+      <h5>{t('Map View')}</h5>
 
       <RadioTiles
         options={[
           {
-            label: 'Default',
-            value: 'roadmap',
+            label: t('Default'),
+            value: t('roadmap'),
```
Comment (Qrtn) [inline src/components/settings/SettingsPage.js:98]: Don't translate any "values".

Comment (Qrtn) [reply src/components/settings/SettingsPage.js:98]: Those have semantic meaning to the API.

File: src/components/settings/SettingsPage.js [hunk 4]
```diff
@@ -52,20 +53,21 @@ const Page = styled.div`
...
-      {!hideTitle && <h2>Settings</h2>}
-      <h3>Viewing Preferences</h3>
+      {!hideTitle && <h2>{t('Settings')}</h2>}
+      <h3>{t('Viewing Preferences')}</h3>
 
       {[
         {
-          field: 'showLabels',
-          label: 'Show Labels',
+          field: t('showLabels'),
```
Comment (Qrtn) [inline src/components/settings/SettingsPage.js:65]: Don't translate any fields.

File: src/components/filter/TreeSelect.js [hunk 5]
```diff
@@ -134,23 +135,26 @@ const TreeSelectContainer = styled.div`
...
+  return (
+    <TreeSelectContainer popover={popover}>
+      <NonrenderTreeSelect
+        data={data}
+        onChange={onChange}
+        texts={{
+          placeholder: t('Select a type...'),
+          inlineSearchPlaceholder: t('Search for a type...'),
+        }}
+        showDropdown={popover ? t('default') : t('always')}
```
Comment (Qrtn) [inline src/components/filter/TreeSelect.js:149]: Don't translate default and always. Those are arguments to the library.

File: src/components/desktop/Header.js [hunk 6]
```diff
@@ -60,21 +61,24 @@ const StyledHeader = styled.header`
...
-)
+const Header = () => {
+  const { t } = useTranslation()
+  return (
+    <StyledHeader>
+      <img src="/logo_white.svg" alt="Falling Fruit Logo" />
+      <nav>
+        <ul>
+          <li className="active"> {t('Map')}</li>
+          <li> {t('About')}</li>
```
Comment (Qrtn) [inline src/components/desktop/Header.js:72]: Remove extra space before these two (not the button):

`<li>{t('Map')}</li>`

`<li>{t('About')}</li>`

**Review by Qrtn — CHANGES_REQUESTED** [review]
Great work!! A few things shouldn't be translated because they're code, not labels.

**Review by Qrtn — APPROVED** [review]
💯💯

**Review by laurenho025 — APPROVED** [review]
looks great!!!

**Pull request merged**